src/cmd/pay.rs

```diff
@@ -28,8 +28,16 @@ pub enum Cmd {
 
 #[derive(Debug, StructOpt)]
 pub struct One {
-    #[structopt(flatten)]
```

Comment:
I don't love having to parse these manually rather than previous structopt flatten, but I think the UI of overloading the <amount> positional param on the command line is worth it. If there is a neater and more idiomatic way of doing that, please let me know!